util/util.go

```diff
@@ -280,7 +280,7 @@ func AtomicallyWriteFile(filename string, data []byte, perm os.FileMode) error {
 		return err
 	}
 
-	if err = f.Chmod(perm); err != nil {
+	if err = os.Chmod(filename, perm); err != nil {
```

Comment:
We want to change permissions on the `TempFile`, `f`, we have created above rather than the `filename` passed to the function:

```suggestion
	if err = os.Chmod(f.Name(), perm); err != nil {
```